### DISCUSSION TITLE
does JAX provide function where we want to change the view of jax Object with the same data

### DISCUSSION THREAD

**Discussion by agaraman0** [jax-ml/jax#16178] [Q&A]
NumPy has something like this 

https://numpy.org/doc/stable/reference/generated/numpy.ndarray.view.html

**Comment by jakevdp** [comment]
No, in JAX it is not possible for two array objects at the Python level to share the same memory buffer.

That said, when JAX code is wrapped in `jit`, the compiler generally optimize array memory layouts, making it unnecessary for the user to manage these kinds of views manually.
